Sources/FaunaDB/Expr.swift

```diff
@@ -55,10 +117,13 @@ public struct Obj: Expr, AsJson, CustomStringConvertible {
         return wrapped.description
     }
 
+    /// Converts an native dictionary to a `Obj` instance.
```

Comment:
bah sorry i see what happened, there were two articles in this sentence.

the first "a" should be "a", but the second should be "an", so:
    /// Converts a native dictionary to an `Obj` instance.